extensions/Worker.Extensions.ServiceBus/src/Proto/settlement.proto

```diff
@@ -19,6 +20,18 @@ service Settlement {
 
   // Defers a message
   rpc Defer (DeferRequest) returns (google.protobuf.Empty) {}
+
+  // Get
+  rpc Get (GetRequest) returns (GetResponse) {}
```

Comment:
This naming is ambiguous as to what it is getting, can we expand upon it?

```suggestion
  rpc GetSession (GetSessionRequest) returns (GetSessionResponse) {}
```

Recommend similar naming changes for the other APIs added in this PR

Reply:
done!